src/scripts/main.js

```diff
@@ -1 +1,80 @@
 'use strict';
+
+async function wait(ms) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+const firstPromise = new Promise(async (resolve, reject) => {
+  let isTrue = false;
+
+  document.addEventListener('click', () => {
+    isTrue = true;
+    resolve('First promise was resolved');
+  });
+
+  await wait(3000);
+
+  if (!isTrue) {
+    reject(new Error('First promise was rejected'));
+  }
+});
```

Comment:
You have an `async` keyword in the executor of your `firstPromise`. The executor function of a Promise should not be an async function. Async functions are designed to work with Promises, but the executor function is a lower-level part of a Promise. You should remove the `async` keyword here.